nexus/external-api/src/lib.rs

```diff
@@ -964,19 +967,19 @@ pub trait NexusExternalApi {
         path = "/v1/system/ip-pools/{pool}",
         tags = ["system/ip-pools"],
     }]
-    async fn ip_pool_update(
+    async fn system_ip_pool_update(
         rqctx: RequestContext<Self::Context>,
         path_params: Path<params::IpPoolPath>,
         updates: TypedBody<params::IpPoolUpdate>,
-    ) -> Result<HttpResponseOk<views::IpPool>, HttpError>;
+    ) -> Result<HttpResponseOk<views::SystemIpPool>, HttpError>;
 
     /// Fetch IP pool utilization
```

Comment:
```suggestion
    /// View IP pool utilization
```

Enh.. we're about 60/40 fetch vs. view so whatever...